models/Movie.model.js

```diff
@@ -0,0 +1,15 @@
+const {Schema, model} = require("mongoose");
+
+const movieSchema = new Schema(
+    {
+        title: String,
+    genre: String,
+    plot: String,
+    cast: [String]
```

Comment:
We must have a reference, that's how the models link:

cast: [{ type: Schema.Types.ObjectId, ref: 'Celebrity' }]